models/_root/cart.yml

```diff
@@ -121,17 +139,33 @@ properties:
           type: string
           description: Unique ID of the coupon.
         image:
-          $ref: ../common/image.yml
+          type: object
+          properties:
+            alt:
+              type: string
+              description: Corresponding text to display for missing images, accessible screen readers, etc.
+            data:
+              type: string
+              description: Internal URL to the image.
         max_purchase_quantity:
-          type: 'null'
+          type: object

```

Comment:
this seems like an integer or string, not an object.

If it is an object, it needs properties.